src/Microsoft.Data/ArrowStringDataFrameColumn.cs

```diff
@@ -469,5 +469,77 @@ protected internal override Delegate GetDataViewGetter(DataViewRowCursor cursor)
 
         private ValueGetter<ReadOnlyMemory<char>> CreateValueGetterDelegate(DataViewRowCursor cursor) =>
             (ref ReadOnlyMemory<char> value) => value = this[cursor.Position].AsMemory();
+
+        public PrimitiveDataFrameColumn<bool> ElementwiseEquals(string value)
+        {
+            ReadOnlySpan<byte> bytes = value != null ? Encoding.UTF8.GetBytes(value) : default(ReadOnlySpan<byte>);
+            PrimitiveDataFrameColumn<bool> ret = new PrimitiveDataFrameColumn<bool>(Name, Length);
+            for (long i = 0; i < Length; i++)
+            {
+                var strBytes = GetBytes(i);
+                if (strBytes.Length != bytes.Length)
```

Comment:
Unfortunately, ReadOnlySpan<T> operator == does not do an element wise value comparison

Reply:
Use `SequenceEquals` instead.

---
In reply to: [341730642](https://github.com/dotnet/corefxlab/pull/2772#discussion_r341730642) [](ancestors = 341730642)